src/bankAccount.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount) {
+  this.name = name;
+  this.amount = amount;
+  this.accountHistory = [`Initial: ${this.amount}`];
+};
+
+BankAccount.prototype.getInfo = function() {
+  return (`Name: ${this.name}, Amount: ${this.amount}$`);
+};
+
+BankAccount.prototype.addMoney = function(sum, description) {
```

Comment:
sum - сумма чисел в английском языке
```suggestion
BankAccount.prototype.addMoney = function(amount, operation) {
```

fix withdraw